File: src/main/java/com/beemer/fromis9server/introduction/model/BannerImages.java
package com.beemer.fromis9server.introduction.model;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Data;

@Data
@Entity
@Table(name = "BannerImages")
public class BannerImages {
    @Id
    @Column(name = "imageUrl", nullable = false)
    private String imageUrl;
}

File: src/main/kotlin/com/beemer/unofficial/fromis9/fromis9/entity/BannerImages.kt
package com.beemer.unofficial.fromis9.fromis9.entity

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.Table

@Entity
@Table(name = "BannerImages")
data class BannerImages(
    @Id
    @Column(name = "imageUrl", nullable = false)
    val imageUrl: String
)
